lib/nodes/timezone.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:async';
+
+import 'package:dslink/dslink.dart';
+import "package:timezone/src/env.dart" as TimezoneEnv;
+
+import 'common.dart';
+
+class TimezoneNode extends SimpleNode {
+  static const String isType = "timezone";
+  static const String pathName = "timezone";
+
+  static Map<String, dynamic> def(String tz) => {
+    r"$name": "Timezone",
+    r"$type": "string",
+    r"$is": isType,
+    "?value": tz,
+    r"$writable": "write"
+  };
+
+  ICalendarLocalSchedule schedule;
+
+  final LinkProvider link;
+  TimezoneNode(String path, this.link) : super(path);
+
+  @override
+  onSetValue(value) {
+    if (value is String) {
+      var loc = const [
+        "UTC",
+        "Etc/GMT"
+      ].contains(value) ? TimezoneEnv.UTC : TimezoneEnv.getLocation(value);
+      if (loc != null) {
+        schedule.timezone = loc;
+        new Future(() {
+          link.save();
+        });
+        schedule.loadSchedule(true);
+        return false;
+      }
+    }
+    return true;
```

Comment:
I'm not sure I understand the use of the boolean return here. What's the meaning?

Reply:
This is part of the API for SimpleNode
> Callback used to accept or reject a value when it is set. Return true to reject the value, and false to accept it.

The logic unfortunately is beyond me as to why that particular return value for that particular meaning, but it's a documented part of the dart-dslink-sdk and would be a breaking change to modify now.